DevEdu.API/DevEdu.API/Configuration/MapperProfile.cs

```diff
@@ -66,7 +66,9 @@ private void CreateMappingFromDto()
             CreateMap<CommentDto, CommentInfoOutputModel>()
                 .ForMember(dest => dest.Date, opt => opt.MapFrom(src => src.Date.ToString(_dateFormat)));
             CreateMap<CourseTopicDto, CourseTopicOutputModel>();
-            CreateMap<CourseDto, CourseInfoBaseOutputModel>();
+            //CreateMap<CourseDto, CourseInfoBaseOutputModel>();
```

Comment:
please remove this line if it is not needed

Reply:
+